src/openlifu/db/photoscan.py

```diff
@@ -0,0 +1,229 @@
+    param model_abspath: absolute filepath to model data
+          texture abspath: absolute filepath to texture data
+    Returns: Photoscan data as (model_vtkpolydata, texture_vtkimagedata)
+    """
+    model_polydata  = load_model(model_abspath)
+    texture_imagedata = load_texture(texture_abspath)
+
+    return (model_polydata, texture_imagedata)
+
+def load_data_from_photoscan(photoscan: Photoscan, parent_dir) -> Tuple[vtk.vtkPolyData, vtk.vtkImageData]:
+    """
+    param parent_dir: parent directory containing model and texture data filepaths
+    Returns: Photoscan data as (model_vtkpolydata, texture_vtkimagedata)
+    """
+    model_polydata  = load_model(Path(parent_dir)/photoscan.model_filename)
+    texture_imagedata = load_texture(Path(parent_dir)/photoscan.texture_filename)
+
+    return (model_polydata, texture_imagedata)
+
+def load_model(file_name):
+    """ This function assumes that the model is saved following the LPS coordinate system."""
+    if not Path(file_name).exists():
+        raise FileNotFoundError(f'Model filepath does not exist: {file_name}')
+    mesh = read_as_vtkpolydata(file_name)
+    mesh_ras = convert_between_ras_and_lps(mesh)
+
+    return mesh_ras
+
+def load_texture(file_name):
+    if not Path(file_name).exists():
+        raise FileNotFoundError(f'Texture data filepath does not exist: {file_name}')
+    texture = read_as_vtkimagedata(file_name)
+    return texture
+
+def read_as_vtkpolydata(file_name):
+
+    valid_suffixes = ['.g', '.obj', '.stl', '.ply', '.vtk', '.vtp']
+    path = Path(file_name)
+    if path.suffix:
+        ext = path.suffix.lower()
+    if path.suffix not in valid_suffixes:
+        raise ValueError(f"File format {path.suffix} not supported by reader")
+    else:
+        if ext == ".ply":
+            reader = vtk.vtkPLYReader()
+            reader.SetFileName(file_name)
+            reader.Update()
+            poly_data = reader.GetOutput()
+        elif ext == ".vtp":
+            reader = vtk.vtkXMLPolyDataReader()
+            reader.SetFileName(file_name)
+            reader.Update()
+            poly_data = reader.GetOutput()
+        elif ext == ".obj":
+            reader = vtk.vtkOBJReader()
+            reader.SetFileName(file_name)
+            reader.Update()
+            poly_data = reader.GetOutput()
+        elif ext == ".stl":
+            reader = vtk.vtkSTLReader()
+            reader.SetFileName(file_name)
+            reader.Update()
+            poly_data = reader.GetOutput()
+        elif ext == ".vtk":
+            reader = vtk.vtkPolyDataReader()
+            reader.SetFileName(file_name)
+            reader.Update()
+            poly_data = reader.GetOutput()
+        elif ext == ".g":
+            reader = vtk.vtkBYUReader()
+            reader.SetGeometryFileName(file_name)
+            reader.Update()
+            poly_data = reader.GetOutput()
+
+        return poly_data
+
+def convert_numpy_to_vtkimage(image_numpy):
```

Comment:
This function could use a docstring and type annotations. What is the expected shape of `image_numpy`? it's only for 2D images right?

I feel this function could use a unit test. The test could create a tiny 2D numpy array with some known values, run it through this function to get a vtk image, and then just make sure that the two agree at some location where the value is known